src/main/webapp/app/quiz/participate/multiple-choice-question/multiple-choice-question.component.ts

```diff
@@ -88,4 +96,21 @@ export class MultipleChoiceQuestionComponent implements OnInit, OnDestroy {
             }) !== -1
         );
     }
+
+    open(content: any) {
```

Comment:
How is this code used?
Can this function and the added import in the `constructor` go away?

Reply:
This is there so the pop-up opens of the warning text

Reply:
Are you sure this is not a left over from refactoring to `QuizScoringInfostudentModalComponent`?